calcom/cal.com
# Installed Apps page revamp

## What does this PR do?

Revamping "Installed Apps" page to just show necessary things.

Fixes #2715
Fixes #2571

Loom Video: https://www.loom.com/share/0870dda91c484e87995a0be26c7f646f
Loom Video follow-up: https://www.loom.com/share/8531837b3eb44506bcd42b9eccdc357d

## Type of change




## How should this be tested?

- Go to "Installed Apps" page and see only installed apps shown
- New empty state would be visible if no installed app is present
- Webhooks were moved to "Developer" tab in Settings along with API Keys (moved from "Security" tab)
- Connect additional calendars section in Installed Apps page was moved to a new dropdown next to default calendar dropdown
